### PR TITLE
Rexfue rev string on minidisplay

### PR DESCRIPTION
**Pull request by rexfue**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
hmm, why did you make a new PR instead of pushing to the existing one?

**Comment by ThomasWaldmann** [comment]
hmm, seems like this is based on the "as is" acceptance / merging of the other one.

so it might be the case that you can improve the other one without getting into the way of this one.

**Comment by rexfue** [comment]
I'm not sure, what the problem was. I hope, i didn't mess it up too much. I couldn't push  my changes. Then I pulled a fresh version and tried to push. And GitHub wanted a new PR. 
I'm still on "Kriegsfuß" with this ;-)
Let's merge, and then all problems are gone - hopefully.
Other question: what is your environment on linux to compile this? I tried platformio on a PI and it worked.

**Comment by ThomasWaldmann** [comment]
I use Debian Linux, git on the commandline and the Arduino IDE for compiling.

**Comment by rexfue** [comment]
OK, I don't like Arduino-IDE because of the ugly editor. On MAC I use Visual Code with Platformio.

Do You agree to merge this?

File: multigeiger/multigeiger.ino [hunk 1]
```diff
@@ -355,33 +355,27 @@ void setup()
...
+// build LoRa software version
+char softvers[8];
+strcpy(softvers,software_version);
+char *p;
+p = strtok(softvers,".");
+lora_software_version[0] = (uint8_t)atoi(p);
+p = strtok(NULL,".");
+lora_software_version[1] = (uint8_t)atoi(p);
+#endif
+
```
Comment (ThomasWaldmann) [inline multigeiger/multigeiger.ino]: is that indented differently than the other code? please only use spaces.

File: multigeiger/multigeiger.ino [hunk 2]
```diff
@@ -713,24 +707,20 @@ void DisplayStartscreen(void){
...
   char rv[9];
-  char delim[] = "_-";
-  char *p; 
-  p = strtok((char *)rvStr,delim); 
-  strcpy(rv, p);
+  sprintf(rv,"V%s",software_version);
   u8x8.drawString(2, 4, rv);
   *rv = '\0';
-  p = strtok(NULL,delim);
+  strcpy(softversdat, software_version_date);
```
Comment (ThomasWaldmann) [inline multigeiger/multigeiger.ino]: sometimes you leave a space after comma (which is good, better readable), but often not.

would be greatly improve readability if you always did.

Comment (ThomasWaldmann) [reply multigeiger/multigeiger.ino]: and can we please keep the "e", so have "date", not "dat"?

File: multigeiger/userdefines-example.h [hunk 3]
```diff
@@ -5,10 +5,12 @@
 // and then edit userdefines.h as appropriate for your use case / hardware.
 //====================================================================================================================================
 
-#ifndef CPU           // if using platformio, CPU is defined in platformio.ini
-// ** select (uncomment) exactly one of following CPU (board type) defines:
-#define CPU WIFI                                // Heltec WiFi Kit 32 (big display, WiFi-only)
-// #define CPU STICK                             // Heltec Wireless Stick (small display, WiFi+LoRa)
+// If you use platformio, CPU is defined in platformio.ini and
+// you don't need to change the definition of CPU
```
Comment (ThomasWaldmann) [inline multigeiger/userdefines-example.h]: Please add a "." at the end of sentence (always).

File: multigeiger/userdefines-example.h [hunk 4]
```diff
@@ -5,10 +5,12 @@
...
+// Arduino-IDE users *must* define CPU here and uncomment one and only one of following CPU-Defines
```
Comment (ThomasWaldmann) [inline multigeiger/userdefines-example.h]: In English, dashes ("Bindestrich") is much more rare than in German.
Usually it is just a blank ("Leerzeichen") instead.

File: multigeiger/multigeiger.ino [hunk 5]
```diff
@@ -917,8 +924,8 @@ void sendData2TTN(int sendwhat, unsigned int hvpulses) {
   // next byte is the tube version
   ttnData[4] = tubes[TUBE_TYPE].nbr;
     // and last is software version
```
Comment (ThomasWaldmann) [inline multigeiger/multigeiger.ino]: not in new code, but there seems to be a bad indentation here.

File: multigeiger/version.h [hunk 6]
```diff
@@ -1,15 +1,18 @@
-// Revision history:
+// To fit in 16bit for lora version number we have
+// some limits here: max. VERSION_MAJOR = 15, max. VERSION_PATCH = 15
+#define VERSION_MAJOR 1
+#define VERSION_MINOR 11
+#define VERSION_PATCH 0
+#define VERSION_DATE "2019-12-16"
```
Comment (ThomasWaldmann) [inline multigeiger/version.h]: We could have a comment above this that this is a date in YYYY-MM-DD format.

It is clear with the current value, but not for every value.

**Comment by ThomasWaldmann** [comment]
OK, guess that is all for now. Review finished.

**Comment by ThomasWaldmann** [comment]
BTW, I do not use the arduino editor.

I switched the IDE to "external editor", use vim and only use the IDE to compile / flash.

File: multigeiger/version.h [hunk 7]
```diff
@@ -1,8 +1,9 @@
 // To fit in 16bit for lora version number we have
-// some limits here: max. VERSION_MAJOR = 15, max. VERSION_PATCH = 15
+// some limits here: max. VERSION_MAJOR = 15, max. VERSION_PATCH = 15.
 #define VERSION_MAJOR 1
 #define VERSION_MINOR 11
 #define VERSION_PATCH 0
+// Date is in format "YYY-MM-DD".
```
Comment (ThomasWaldmann) [inline multigeiger/version.h]: one Y more please.

Comment (rexfue) [reply multigeiger/version.h]: done

**Pull request merged**